control/lti.py

```diff
@@ -203,6 +211,26 @@ def ispassive(self):
         from control.passivity import ispassive
         return ispassive(self)
 
+    # convenience aliases
+    # most function are only forward declaraed and patched in the __init__.py to avoid circular imports
+
+    # conversions
+    to_ss: Callable
+    to_tf: Callable
+
+    # system interconnections
+    feedback: Callable
+
+    # freq domain plotting
+    bode_plot: Callable
+    nyquist_plot: Callable
+    nichols_plot: Callable
+
+    # time domain simulation
+    forced_response = control.timeresp.forced_response
+    impulse_response = control.timeresp.impulse_response
+    step_response = control.timeresp.step_response
```

Comment:
We should add (short) docstrings for these.  Since these are initialized as assigned member variables instead of standard function (using `def`), we will have to use the doc-comment format (lines starting with #: before the variable).  See `InputOutputSystem.nstates` for an example.

I think these can be short, perhaps of the form: "Bode plot; see `bode_plot` function for more information".